assets/js/workflow-diagram/WorkflowDiagram.tsx

```diff
@@ -35,6 +35,8 @@ type WorkflowDiagramProps = {
   selection: string | null;
   onSelectionChange: (id: string | null) => void;
   forceFit?: boolean;
+  showAiAssistant?: boolean;
+  aiAssistantId?: string;
```

Comment:
If `aiAssistantId` is only set when `showAiAssistant` is true then we can drop one of the props and use just `aiAssistantId` and then derive show from it.